Add "Earth Retirement" (#5007)
endless-sky/endless-sky: data/human/transport missions.txt

			`The Testament enters the planet's atmosphere at high speed and lands somewhat uncontrolled near a waste processing plant. The cargo hold is glowing red hot and some areas in the cargo hold appear to be on fire.`
			`	Veronica disembarks, walks over to you, and apologizes for the difficult trip. She explains that the cooling systems failed, causing the sushi calamity you experienced during the multiple legs of your journey.`
			`	You are paid <payment> and she asks you not to talk about this escort job to anybody. You leave the docks quickly in order to escape the horrible smell of overheated fish that is coming from Veronica and her ship.`

mission "Earth Retirement"
	name "Retired Family to Earth"

> 	description "Transport two seniors to <destination>, which is where they plan to retire. They can only promise you <payment>"

Reviewer: ```suggestion
	description "Transport two seniors to <destination>, which is where they plan to retire. They can only promise you <payment>."
```

> 	description "Transport two seniors to <destination>, which is where they plan to retire. They can only promise you <payment>."

Reviewer 2: ```suggestion
	description "Transport two seniors to <destination>, where they plan to retire. They can only promise you <payment>."
```

	minor
	source
		attributes paradise
	destination "Earth"
	to offer
		random < 15
	passengers 2
	
	on offer
		conversation
			`As you wander through the spaceport, you are see an elderly man and woman sitting on a bench. They both seem to be talking about something that is causing them some amount of stress.`
			choice

> 				`	"Yes. Who are you two?"`

Reviewer 3: ```suggestion
				`	"Do you need any help?"`
```
IMO, these two responses fit more in a situation where you're being approached by the mission offerer rather than a situation where you're approaching the mission offerer (as it appears to be here).

> 				`	"Yes. Who are you two?"`

Reviewer 2: ```suggestion
				`	"Who are you two?"`
```
`Yes` seems to be a strange thing to open a conversation with.

> 				`	"I do own a ship, but it's not taking passengers right now."`

Reviewer 3: ```suggestion
				`	(Ignore them.)`
```

					decline

> 			`	“My name is Donna, and his name is Charles," she says. "We were yard workers here on <origin> for nearly fifty years, but we're both starting to get too old for our jobs. So, we've made the hard decision to retire. We've already chosen a location on the best place we could afford, which... well, it happened to be on Earth, so Charles and I are looking for someone to transport us.”`

Reviewer 3: ```suggestion
			`	“My name is Donna, and his name is Charles," she says. "We were yard workers here on <origin> for nearly fifty years, but we're both starting to get too old for our jobs. So, we've made the hard decision to retire. We've already chosen a location on the best place we could afford, which... well, happened to be on Earth, so Charles and I are looking for someone to transport us.”`
```

> 			`	“My name is Donna, and his name is Charles" she says. "We were yard workers here on <source> for nearly fifty years, but we're both starting to get too old for our jobs. So, we've made the hard decision to retire. We've already chosen a location on the best place we could afford, which... well, it happened to be on Earth, so Charles and I are looking for someone to transport us.”`

Reviewer: ```suggestion
			`	“My name is Donna, and his name is Charles," she says. "We were yard workers here on <origin> for nearly fifty years, but we're both starting to get too old for our jobs. So, we've made the hard decision to retire. We've already chosen a location on the best place we could afford, which... well, it happened to be on Earth, so Charles and I are looking for someone to transport us.”`
```

> 			`	“My name is Donna, and his name is Charles" she says. "We were yard workers here on <source> for nearly fifty years, but we're both starting to get too old for our jobs. So, we've made the hard desicion to retire. We've already chose a location on the best place we could afford, which... well, it happened to be on Earth, so me and Charles are looking for a someone to transport us.”`

Reviewer 4: ```suggestion
			`	“My name is Donna, and his name is Charles" she says. "We were yard workers here on <source> for nearly fifty years, but we're both starting to get too old for our jobs. So, we've made the hard decision to retire. We've already chosen a location on the best place we could afford, which... well, it happened to be on Earth, so Charles and I are looking for someone to transport us.”`
```

> 			`	As Donna talks, Charles begins to frown. “I supposed you could put it like that.”`

Reviewer 3: ```suggestion
			`	As Donna talks, Charles begins to frown. “I suppose you could put it like that.”`
```

			`	“Is that wrong?” Donna replies, who appears to be bracing for the response.`
			`	The man's look of resignation starts to turn to one of anger. "Of course it's wrong. We've spent our whole lives working for these spoiled brats, and what do we get in return? We get dumped on Earth like we're dead weight!"`

> 			`	"I know you're upset about this," interrupts Donna "but just getting angry isn't going to do anything!" Charles remains silent for a long time, and then mutters a reluctant apology.`

Reviewer 2: ```suggestion
			`	"I know you're upset about this," interrupts Donna, "but just getting angry isn't going to do anything!" Charles remains silent for a long time, and then mutters a reluctant apology.`
```

			`	Donna talks to you again. "I'm sorry about that. As you can tell, we're under a lot of stress right now. Can you help us? We'll pay you <payment> if you do."`
			choice
				`	"I can handle two passengers."`

> 					accept

Reviewer: ```suggestion
```

				`	"I'm sorry, but I'm not headed to Earth right now."`
					decline

> 			`	You show Charles and Donna to their bunks, and they both thank you in advance.`

Reviewer: ```suggestion
			`	You show Charles and Donna to their bunks, and they both thank you in advance.`
				accept
```

	
	on visit
		dialog phrase "generic passenger on visit"
	on complete
		conversation

> 			`Once you land on Earth, Donna and Charles pick up their belongings and start to leave your ship. Before he leaves Charles stops by you, and takes out a large clump of credit chips with a total value of <payment>. "Here you go." he mutters. "I hope your life goes better than ours has."`

Reviewer: ```suggestion
			`Once you land on Earth, Donna and Charles pick up their belongings and start to leave your ship. Before he leaves Charles stops by you, and takes out a large clump of credit chips with a total value of <payment>. "Here you go," he mutters. "I hope your life goes better than ours has."`
```

> 			`Once you land on Earth, Donna and Charles pick up their belongings and start to leave your ship. Charles stops by you, and takes out a large clump of credit chips with a total value of <payment>. "Here you go.", he mutters, "I hope your life goes better than ours has."`

Reviewer 3: ```suggestion
			`Once you land on Earth, Donna and Charles pick up their belongings and start to leave your ship. Before he leaves, Charles stops by you, and takes out a large clump of credit chips with a total value of <payment>. "Here you go.", he mutters, "I hope your life goes better than ours has."`
```

> 			`Once you land on Earth, Donna and Charles pick up their belongings and start to leave your ship. Charles stops by you, and takes out a large clump of credit chips with a total value of <payment>. "Here you go.", he mutters, "I hope your life goes better than ours has."`

Reviewer 2: ```suggestion
			`Once you land on Earth, Donna and Charles pick up their belongings and start to leave your ship. Charles stops by you and takes out a large clump of credit chips with a total value of <payment>. "Here you go," he mutters. "I hope your life goes better than ours has."`
```

> 			`Once you land on Earth, Donna and Charles both pick up their belongings and start to leave your ship. Charles stops by you, and takes out a large clump of credit chips with a total value of <payment>. "Here you go.", he mutters, "I hope your life goes better than ours has."`

Reviewer 4: ```suggestion
			`Once you land on Earth, Donna and Charles pick up their belongings and start to leave your ship. Charles stops by you, and takes out a large clump of credit chips with a total value of <payment>. "Here you go.", he mutters, "I hope your life goes better than ours has."`
```

			choice
				`	"Thank you. I hope you have the best of luck on Earth."`
					goto payment

> 				`	"You can keep those. You sound like you needed them more than I do."`

Reviewer 2: ```suggestion
				`	"You can keep those. You sound like you need them more than I do."`
```
`need` seems a little more natural, but this is just personal preference.

					goto reject
			
			label payment
			`	They slowly make their way to the train station. Presumably, their new apartment is far from here.`
			payment

> 			payment 6000

Reviewer: Do note that even with #5157 (at least in its current state), a conversation payment can not be based off of the distance you had to travel. Therefore, this first `payment` line does nothing.

> 			`	You wish the couple the best of luck on Earth, and they slowly make their way to the train station. Presumably, their new apartment is far from here.`

Reviewer 2: ```suggestion
			`	They slowly make their way to the train station. Presumably, their new apartment is far from here.`
```
You already mention the `best of luck` bit in dialogue previous to this, so repeating it is unnecessary.

			label reject

> 			`	Charles looks surprised for a moment, and then, for the first time you've seen, he smiles. "Thank you, Captain <last>. I wish there were more people like you in the paradise worlds."`

Reviewer 2: ```suggestion
			`	Charles looks surprised for a moment, and then, for the first time you've seen, he smiles. "Thank you, Captain <last>. I wish there were more people like you in the Paradise Worlds."`
```

> 			`	Charles looks suprised for a moment, and then, for the first time you've seen, he smiles. "Thank you, Captain <last>. I wish there where more people like you in the paradise worlds."`

Reviewer 4: ```suggestion
			`	Charles looks surprised for a moment, and then, for the first time you've seen, he smiles. "Thank you, Captain <last>. I wish there were more people like you in the paradise worlds."`
```
